test_pull_requests

```diff
@@ -1084,8 +1206,7 @@ popd
           #    - NOTFOUND
 
           # Capture the build_url from the regex match
-          build_url = $1
-          extended_tests = $2
+          _, build_url, base_commit, extended_tests = extract_bot_comment_metadata(comment, settings)
```

Comment:
I agree, I should be parsing out the metadata and state info before the
case. Will look at this tomorrow

On Aug 9, 2016 4:57 PM, "Dan McPherson" <notifications@github.com> wrote:

> In test_pull_requests
> <https://github.com/openshift/test-pull-requests/pull/31#discussion_r74142798>
> :
>
> > @@ -1084,8 +1206,7 @@ popd
> >            #    - NOTFOUND
> >
> >            # Capture the build_url from the regex match
> > -          build_url = $1
> > -          extended_tests = $2
> > +          _, build_url, base_commit, extended_tests = extract_bot_comment_metadata(comment, settings)
>
> It's weird to be in a case statement for a regex and reparse one of the
> values the outer regex is already capturing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/test-pull-requests/pull/31/files/22d78344f24bcb8fabe8555cd6bfdf5850679864#r74142798>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABd0FP-Oa_cnZmgpunECEPUWYaBia9rLks5qeOmtgaJpZM4JcBHJ>
> .
>